Update connector_resource docs (#34)
fivetran/terraform-provider-fivetran: docs/resources/connector.md

}
```

### NOTE: resources indirect dependencies

> As you may notice the connector resource takes group_id parameter value from group resource, but the destination resource is also depend on group. And once you'll try to destroy destination resource infrastructure terraform plan will be created successfully, but it return an error on apply because Fivetran API doesn't allow you to delete destination that have linked connectors. To solve this problem you should define explicit depends_on between the connector and destination:

Reviewer: ```suggestion
The connector resource receives the `group_id` parameter value from the group resource, but the destination resource depends on the group resource.  Once you try to destroy the destination resource infrastructure, the terraform plan is created successfully, but it returns an error on apply because the Fivetran API doesn't allow you to delete destinations that have linked connectors. To solve this problem, you should explicitly define `depends_on` between the connector and destination:
```

Reviewer: What do you mean by "on apply"? When you click an Apply button, or apply using a command?

Author: no, it means when you are running `terraform apply` command in CLI

Author: It's terraform specific flow. Once you've prepared configuration you run `terraform plan` command and it checks current state and provides you information about what changes will be. If all is OK you should run the next command `terraform apply` to effectively apply changes on infrastructure

Author: This situation is about the case when you've deleted destination from your config file and trying to apply new config.
`terraform plan` command returns OK status, but `terraform apply` fails unexpectedly

> The connector resource receives the `group_id` parameter value from the group resource, but the destination resource depends on the group resource.  Once you try to destroy the destination resource infrastructure, the terraform plan is created successfully, but it returns an error on apply because the Fivetran API doesn't allow you to delete destinations that have linked connectors. To solve this problem, you should either explicitly define `depends_on` between the connector and destination:

Reviewer: ```suggestion
The connector resource receives the `group_id` parameter value from the group resource, but the destination resource depends on the group resource.  When you try to destroy the destination resource infrastructure, the terraform plan is created successfully, but once you run the `terraform apply` command, it returns an error because the Fivetran API doesn't let you delete destinations that have linked connectors. To solve this problem, you should either explicitly define `depends_on` between the connector and destination:
```

```hcl
resource "fivetran_connector" "amplitude" {
    ...
    depends_on = [
        fivetran_destination.my_destination
    ]
}
```

> or get the group id from destination:

Reviewer: ```suggestion
or get the group ID from destination:
```

> or get the group ID from destination:

Reviewer: ```suggestion
or get the group ID from the destination:
```
